doc-Installing_on_OpenShift_Container_Platform/topics/configuring_external_auth_ocp.adoc

```diff
@@ -0,0 +1,278 @@
+--add-file=/etc/openldap/cacerts/seconday-directory-cert.pem \
+--output=/tmp/updated-auth-configmap.yaml
+----
+
+.Adding target files from different source directories
+
+----
+$ httpd_configmap_generator update \
+--input=/tmp/original-auth-configmap.yaml                                        \
+--add-file=/tmp/uploaded-cert1,/etc/openldap/cacerts/primary-directory-cert.pem  \
+--add-file=/tmp/uploaded-cert2,/etc/openldap/cacerts/seconday-directory-cert.pem \
+--output=/tmp/updated-auth-configmap.yaml
+----
+
+The file ownership and permissions are based on the source files specified;
+in this case the ownership and permissions of the `/tmp/uploaded-cert1`
+and `/tmp/uploaded-cert2` files will be used.
+
+.Adding a target file with user-specified ownership and mode
+
+----
+$ httpd_configmap_generator update \
+--input=/tmp/original-auth-configmap.yaml                          \
+--add-file=/tmp/secondary-keytab,/etc/http2.keytab,600:apache:root \
+--output=/tmp/updated-auth-configmap.yaml
+----
+
+.Adding files by URL
+
+----
+$ httpd_configmap_generator update \
+--input=/tmp/original-auth-configmap.yaml \
+--add-file=http://aab-keycloak:8080/auth/realms/testrealm/protocol/saml/description,/etc/httpd/saml2/idp-metadata.xml,644:root:root \
+--output=/tmp/updated-auth-configmap.yaml
+----
+
+When downloading a file by URL, a target file path and file ownership/mode must be specified.
+
+
+==== Exporting a File from an `authconfig` Map
+
+With the `export` subcommand, you can export a file from the configuration
+map. For example, to extract the `sssd.conf` file from the `authconfig` map:
+
+----
+$ httpd_configmap_generator export \
+--input=/tmp/external-ipa.yaml \
+--file=/etc/sssd/sssd.conf     \
+--output=/tmp/sssd.conf
+----
+
+==== Building the `httpd_configmap_generator` in a Container
+
+The `httpd_configmap_generator` is the container for configuring external authentication for the `httpd` auth pod.
+It is based on the auth `httpd` container and generates the `httpd` `authconfig` map needed to enable external authentication.
+
+Two templates are required to run the `httpd-configmap-generator` application (`httpd-configmap-generator-htmplate.yaml` and `httpd-scc-sysadmin.yaml`), which are available from the https://access.redhat.com/containers/?tab=images&platform=openshift#/registry.access.redhat.com/cloudforms46/cfme-httpd-configmap-generator[Red Hat Container Catalog].
+
+===== Preparing to Deploy the `httpd-configmap-generator` Application
+
+. To obtain the latest `cfme-httpd-configmap-generator` image from the Red Hat Container Catalog, run:
++
+----
+$ oc import-image my-cloudforms46/cfme-httpd-configmap-generator --from=registry.access.redhat.com/cloudforms46/cfme-httpd-configmap-generator --confirm
+----
++
+. The `httpd-configmap-generator` service account must be added to the `httpd-scc-sysadmin` SCC before the `httpd-configmap-generator` can run. To edit the SCC, log in to OpenShift as an admin user: 
```

Comment:
I'd expand SCC in the first instance: security context constraints (SCC)?

Reply:
Good idea! Done in BZ#1540641-addl